updater/update-binary

```diff
@@ -231,9 +232,11 @@ ui_print "|---------------------------------------------|";
 ui_print "|---------------------------------------------|";
 ui_print "|---------------------------------------------|";
 ui_print "|                  Thank You                  |";
-ui_print "|   Reboot to PitchBlack Recovery Manually    |";
+if [[ ! -f "/sbin/keycheck" ]]; then
```

Comment:
Kindly edit and use single [ ] , tab as indent (no spaces), upon newly formatted update-binary.